### ISSUE TITLE
KWin doesn't remember window sizes when Tiling Extension is on, but tiling is off

### ISSUE THREAD

**Issue by telebiker** [kwin-scripts/kwin-tiling#196]
Steps to reproduce:

1. Install Tiling Extension in KWin scripts and enable it
2. Ensure that Tiling Extension works properly
3. Disable tiling using the hotkey (by default is Meta+Shift+F11) (or configure Tiling Extension, unchecking "Enable tiling by default" and reload KDE)
4. Open any window, resize it, move or maximize
5. Minimize that window and open it back

Expected behaviour: window size and position is saved

Actual behaviour: window size and position is not saved, it looks like just opened

```
$ plasmashell --version
plasmashell 5.17.4
$ kf5-config --version
Qt: 5.14.0
KDE Frameworks: 5.65.0
kf5-config: 1.0
```

**Comment by faho** [comment]
Yeah, I think there's a path where we don't check whether we're active.

**Comment by faho** [comment]
Should be fixed via 7d7444d485d8a1d36fecc4e8b195d2c5c7621082
